Tests/check_release_notes.py

```diff
@@ -0,0 +1,6 @@
+import sys
+from pathlib import Path
+
+for rst in Path("docs/releasenotes/").rglob("[1-9]*.rst"):
```

Comment:
```suggestion
for rst in Path("docs/releasenotes").glob("[1-9]*.rst"):
```
I don't think the trailing slash is necessary, and `docs/releasenotes` doesn't have subdirectories.